prisma/seed.ts

```diff
@@ -31,6 +73,7 @@ async function main() {
       isMember: true,
       centerAffiliation: "Nepal",
       photo: createAvatar(),
+      centreId: 1,
```

Comment:
i think it will be better if we keep the id returned by above queries directly (centreId: usCentre.id).Deleting database data without resetting the pk sequence would not reset the primary key sequence. And might cause some issues for new devs. What do you think ?

Reply:
Yes, agree